controller/src/main/java/org/apache/rocketmq/controller/impl/manager/ReplicasInfoManager.java

```diff
@@ -373,6 +374,19 @@ public ControllerResult<Void> getSyncStateData(final List<String> brokerNames) {
         return result;
     }
 
+    public ControllerResult<Void> cleanBrokerData(final CleanControllerBrokerDataRequestHeader requestHeader) {
+        final ControllerResult<Void> result = new ControllerResult<>();
+        String brokerName = requestHeader.getBrokerName();
+        String brokerAddress = requestHeader.getBrokerAddress();
```

Comment:
Multiple brokerAddress are not handled here, eg: 192.168.0.1:30911;192.168.0.2:30911